htgr/triso_fuel/tests

```diff
@@ -1,12 +1,39 @@
 [Tests]
-  [triso_bison]
+  [triso_1d_bison]
     type = CSVDiff
     input = 'triso_1d.i'
     csvdiff = 'triso_1d_out.csv'
     executable_pattern = 'bison*'
     # see #114 on vtb
     rel_err = '1e-3'
   []
+  [triso_2d_aspherical_bison]
+    type = CSVDiff
+    input = 'triso_2d_aspherical.i'
+    csvdiff = 'triso_2d_aspherical_out.csv'
+    executable_pattern = 'bison*'
+    # see #236 on vtb
+    rel_err = '1e-3'
```

Comment:
```suggestion
```
I would rather we try with tight tolerances first. Then if they dont work we can relax

Reply:
Good point.  Not sure why used that before,  just copied and pasted here.  Will change it tighter.